crates/grafbase-local-backend/src/dev.rs

```diff
@@ -109,40 +126,46 @@ pub async fn start(
         hot_reload(
             reload_sender,
             hot_reload_ready_sender,
-            graph_ref,
             subgraph_cache,
             gateway_config_path,
             graph_overrides_path,
+            dev_configuration,
         )
         .await;
     });
 
-    serve(server_config, CliRuntime { ready_sender })
-        .await
-        .map_err(BackendError::Serve)?;
+    let home_dir = dirs::home_dir().ok_or(BackendError::HomeDirectory)?;
+
+    serve(
+        server_config,
+        CliRuntime {
+            ready_sender,
+            port,
+            home_dir,
+        },
+    )
+    .await
+    .map_err(BackendError::Serve)?;
 
     Ok(())
 }
 
-// temporary output handler for internal testing until we move output to the CLI and use a proper terminal crate.
-// none of us uses Windows, right?
-fn output_handler(mut receiver: Receiver<String>) {
-    // gray
-    println!("\x1b[90mWarning: This command is in beta, expect missing features, bugs or breaking changes\x1b[0m\n");
+fn output_handler(mut receiver: Receiver<String>) -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Currently leaving this as an internal handler since this will be merged into the normal CLI